studio-docs/source/explorer/sandbox.mdx

```diff
@@ -76,6 +78,17 @@ Select any accessible organization, graph, and variant to check against and clic
 
 To view the result's _full_ details in Studio, click **Go to details**.
 
+#### Subgraph checks
+
+You can run subgraph checks on federated graphs. If you are introspecting a subgraph you will see a subgraph dropdown where you can select the subgraph that you want your local subgraph to replace.
```

Comment:
"you are" -> "you're"

"you will see a subgraph dropdown...." -> "you can use the subgraph dropdown to select which subgraph you want to check your changes against."